examples/plot_employee_salaries.py

```diff
@@ -0,0 +1,106 @@
+"""
+Predicting the salary of employees
+==================================
+
+Now that we understand how the similarity encoder works, let's 
+see how this can affect prediction performance.
+
+The structure of this example will run as follows:
+* in the employee_salaries dataset we will first create a learning problem by 
+ chosing a column to predict
+* then we will benchmark the different kind of encodings (one-hot, similarity encoding, target encoding) 
+by fitting the target with their respective output with a Ridge Regression
+
+
+"""
+
+#########################################################################
+# 
+import os
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.preprocessing import CategoricalEncoder
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import RidgeCV
+from sklearn.model_selection import KFold
+from sklearn.model_selection import cross_val_score
+from sklearn.compose import ColumnTransformer
```

Comment:
You should mention in bold in the text of the example that it requires Scikit-learn master